pkg/util/config/config.go

```diff
@@ -123,3 +129,13 @@ func Copy(c *Config) Config {
 
 	return cfg
 }
+
+func validate(cfg *Config) (err error) {
+	if cfg.MinInstances > 0 && cfg.MaxInstances > 0 && cfg.MinInstances > cfg.MaxInstances {
+		err = fmt.Errorf("minimum number of instances %d is set higher than the maximum number %d")
+	}
+	if cfg.Workers == 0 {
```

Comment:
cfg.Workers > 0 ? It may be tempting to supply -1 meaning no limit :)

Reply:
it's unit32.